azurerm/internal/tools/website-scaffold/main.go

```diff
@@ -537,14 +537,6 @@ func (gen documentationGenerator) buildDescriptionForArgument(name string, field
 	}
 
 	if field.Elem != nil {
-		if _, ok := field.Elem.(*schema.Schema); ok {
-			if gen.blockIsBefore(name, blockName) {
-				return fmt.Sprintf("A `%s` block as defined above.", name)
-			} else {
-				return fmt.Sprintf("A `%s` block as defined below.", name)
-			}
-		}
```

Comment:
As all the cases for block has been caught in code above, i.e.

```go
	if field.Elem != nil {
		if _, ok := field.Elem.(*schema.Resource); ok {
```

So when we get here, it means this is not a block, so we don't need to check the order since all the document of current attribute (e.g. a list) is covered here. Or maybe I'm wrong?